index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-3">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-4">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-5">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-6">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  </section>
+
+  <!-- Section About -->
+  <section class="about back_dark">        
+    <div class="about_container">
+      <div class="about_text">
+        <h3 class="Title1 Tittle_about">About me</h3>
+        <h4 class="Simpletext">Hello I'm a software developer! I can help you build a product, feauture or website. Look through some of my work and expererience!. If you like what you see and have a prject you need coded, don't hesitate to contact me.</h4>
```

Comment:
Please consider using a `p tag` rather than `h4 tag` to meet the semantic requirements. It will make your portfolio stand out and look more professional 💯 

Moreover, consider using the same set of styles for it as the figma template has used. I mean the same `font-weight, font-size, line-height, font-family, width, and height properties` and others to meet the Figma template.

Reply:
Done. Thank you @ABDULALI3468